ansible_rulebook/app.py

```diff
@@ -234,7 +234,10 @@ def validate_actions(startup_args: StartupArgs) -> None:
     for ruleset in startup_args.rulesets:
         for rule in ruleset.rules:
             for action in rule.actions:
-                if action.action == "run_job_template":
+                if action.action in [
```

Comment:
@akira6592 Can you add a CONTROLLER_ACTIONS constant similar to this https://github.com/ansible/ansible-rulebook/blob/0da53a65379f0ae1128227c4419f9576de83c1c9/ansible_rulebook/app.py#L59 and use it in both the places.

Reply:
@mkanoor Thank you for suggestion. I fixed it.